app/views/users/show.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="container-lg" style="margin-top: 20px">
```

Comment:
It's usually better practice to avoid inline styles whenever possible. Bootstrap has [margin utility classes](https://getbootstrap.com/docs/4.6/utilities/spacing/#examples) `mt-1`..`mt-5` for margin-top. If you need want margin-top: 20px, I might suggest creating your own class named after bootstrap naming conventions `mt-20`.